src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFn) {
```

Comment:
The use of `[].__proto__.sort2` is not recommended. Instead, you should directly extend the `Array.prototype` like this: `Array.prototype.sort2`.